src/relay.js

```diff
@@ -185,6 +184,18 @@ class Relay extends Server {
       autoInitPlayer: false
     }
 
+    if (this.options.allowMultiplePlayers) {
+      options.username = ds.profile.xuid
+
+      let oldMsaResponse = () => {}
+      if (options.onMsaCode) oldMsaResponse = options.onMsaCode.valueOf()
```

Comment:
why .valueOf()?

Reply:
Because wouldn't `oldMsaResponse` change when I set it to kick the player with the login info? Or am I confused